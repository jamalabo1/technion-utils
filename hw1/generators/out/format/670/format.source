A W 453 1
C Y 351 2
D X 534 3
A W 45 4
C Y 80 5
D X 373 6
C W 730 7
C X 64 8
D Z 454 9
B W 850 10
B X 550 11
D W 858 12
C W 652 13
A Z 216 14
C X 749 15
B Z 602 16
C Z 492 17
A X 813 18
B Y 13 19
C X 685 20
C Z 543 21
C Z 509 22
C Z 242 23
A Z 300 24
D Z 558 25
D W 497 26
B W 326 27
D Y 839 28
B W 53 29
B Z 852 30
A Y 720 31
C Z 352 32
A X 748 33
D Z 599 34
B Y 820 35
C X 793 36
A Z 194 37
A Y 787 38
C W 480 39
D Y 319 40
C Y 817 41
C Z 288 42
B Z 61 43
B Y 155 44
A Y 564 45
D Y 372 46
C Z 663 47
D W 147 48
B Y 338 49
A X 896 50
A Z 847 51
D Y 987 52
D W 574 53
B X 394 54
C Y 53 55
B X 427 56
D Z 625 57
B Z 537 58
A W 656 59
B X 583 60
A Y 691 61
C Y 823 62
B Y 196 63
A W 295 64
D W 711 65
C Y 568 66
D Y 587 67
D W 755 68
D W 299 69
C Y 29 70
A Z 594 71
A X 195 72
C X 890 73
A W 745 74
B W 384 75
D X 97 76