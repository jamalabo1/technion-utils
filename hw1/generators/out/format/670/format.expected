Blockchain Info:
1
Sender Name: D
Receiver Name: X
Transaction Value: 97
Transaction timestamp: 76
2
Sender Name: B
Receiver Name: W
Transaction Value: 384
Transaction timestamp: 75
3
Sender Name: A
Receiver Name: W
Transaction Value: 745
Transaction timestamp: 74
4
Sender Name: C
Receiver Name: X
Transaction Value: 890
Transaction timestamp: 73
5
Sender Name: A
Receiver Name: X
Transaction Value: 195
Transaction timestamp: 72
6
Sender Name: A
Receiver Name: Z
Transaction Value: 594
Transaction timestamp: 71
7
Sender Name: C
Receiver Name: Y
Transaction Value: 29
Transaction timestamp: 70
8
Sender Name: D
Receiver Name: W
Transaction Value: 299
Transaction timestamp: 69
9
Sender Name: D
Receiver Name: W
Transaction Value: 755
Transaction timestamp: 68
10
Sender Name: D
Receiver Name: Y
Transaction Value: 587
Transaction timestamp: 67
11
Sender Name: C
Receiver Name: Y
Transaction Value: 568
Transaction timestamp: 66
12
Sender Name: D
Receiver Name: W
Transaction Value: 711
Transaction timestamp: 65
13
Sender Name: A
Receiver Name: W
Transaction Value: 295
Transaction timestamp: 64
14
Sender Name: B
Receiver Name: Y
Transaction Value: 196
Transaction timestamp: 63
15
Sender Name: C
Receiver Name: Y
Transaction Value: 823
Transaction timestamp: 62
16
Sender Name: A
Receiver Name: Y
Transaction Value: 691
Transaction timestamp: 61
17
Sender Name: B
Receiver Name: X
Transaction Value: 583
Transaction timestamp: 60
18
Sender Name: A
Receiver Name: W
Transaction Value: 656
Transaction timestamp: 59
19
Sender Name: B
Receiver Name: Z
Transaction Value: 537
Transaction timestamp: 58
20
Sender Name: D
Receiver Name: Z
Transaction Value: 625
Transaction timestamp: 57
21
Sender Name: B
Receiver Name: X
Transaction Value: 427
Transaction timestamp: 56
22
Sender Name: C
Receiver Name: Y
Transaction Value: 53
Transaction timestamp: 55
23
Sender Name: B
Receiver Name: X
Transaction Value: 394
Transaction timestamp: 54
24
Sender Name: D
Receiver Name: W
Transaction Value: 574
Transaction timestamp: 53
25
Sender Name: D
Receiver Name: Y
Transaction Value: 987
Transaction timestamp: 52
26
Sender Name: A
Receiver Name: Z
Transaction Value: 847
Transaction timestamp: 51
27
Sender Name: A
Receiver Name: X
Transaction Value: 896
Transaction timestamp: 50
28
Sender Name: B
Receiver Name: Y
Transaction Value: 338
Transaction timestamp: 49
29
Sender Name: D
Receiver Name: W
Transaction Value: 147
Transaction timestamp: 48
30
Sender Name: C
Receiver Name: Z
Transaction Value: 663
Transaction timestamp: 47
31
Sender Name: D
Receiver Name: Y
Transaction Value: 372
Transaction timestamp: 46
32
Sender Name: A
Receiver Name: Y
Transaction Value: 564
Transaction timestamp: 45
33
Sender Name: B
Receiver Name: Y
Transaction Value: 155
Transaction timestamp: 44
34
Sender Name: B
Receiver Name: Z
Transaction Value: 61
Transaction timestamp: 43
35
Sender Name: C
Receiver Name: Z
Transaction Value: 288
Transaction timestamp: 42
36
Sender Name: C
Receiver Name: Y
Transaction Value: 817
Transaction timestamp: 41
37
Sender Name: D
Receiver Name: Y
Transaction Value: 319
Transaction timestamp: 40
38
Sender Name: C
Receiver Name: W
Transaction Value: 480
Transaction timestamp: 39
39
Sender Name: A
Receiver Name: Y
Transaction Value: 787
Transaction timestamp: 38
40
Sender Name: A
Receiver Name: Z
Transaction Value: 194
Transaction timestamp: 37
41
Sender Name: C
Receiver Name: X
Transaction Value: 793
Transaction timestamp: 36
42
Sender Name: B
Receiver Name: Y
Transaction Value: 820
Transaction timestamp: 35
43
Sender Name: D
Receiver Name: Z
Transaction Value: 599
Transaction timestamp: 34
44
Sender Name: A
Receiver Name: X
Transaction Value: 748
Transaction timestamp: 33
45
Sender Name: C
Receiver Name: Z
Transaction Value: 352
Transaction timestamp: 32
46
Sender Name: A
Receiver Name: Y
Transaction Value: 720
Transaction timestamp: 31
47
Sender Name: B
Receiver Name: Z
Transaction Value: 852
Transaction timestamp: 30
48
Sender Name: B
Receiver Name: W
Transaction Value: 53
Transaction timestamp: 29
49
Sender Name: D
Receiver Name: Y
Transaction Value: 839
Transaction timestamp: 28
50
Sender Name: B
Receiver Name: W
Transaction Value: 326
Transaction timestamp: 27
51
Sender Name: D
Receiver Name: W
Transaction Value: 497
Transaction timestamp: 26
52
Sender Name: D
Receiver Name: Z
Transaction Value: 558
Transaction timestamp: 25
53
Sender Name: A
Receiver Name: Z
Transaction Value: 300
Transaction timestamp: 24
54
Sender Name: C
Receiver Name: Z
Transaction Value: 242
Transaction timestamp: 23
55
Sender Name: C
Receiver Name: Z
Transaction Value: 509
Transaction timestamp: 22
56
Sender Name: C
Receiver Name: Z
Transaction Value: 543
Transaction timestamp: 21
57
Sender Name: C
Receiver Name: X
Transaction Value: 685
Transaction timestamp: 20
58
Sender Name: B
Receiver Name: Y
Transaction Value: 13
Transaction timestamp: 19
59
Sender Name: A
Receiver Name: X
Transaction Value: 813
Transaction timestamp: 18
60
Sender Name: C
Receiver Name: Z
Transaction Value: 492
Transaction timestamp: 17
61
Sender Name: B
Receiver Name: Z
Transaction Value: 602
Transaction timestamp: 16
62
Sender Name: C
Receiver Name: X
Transaction Value: 749
Transaction timestamp: 15
63
Sender Name: A
Receiver Name: Z
Transaction Value: 216
Transaction timestamp: 14
64
Sender Name: C
Receiver Name: W
Transaction Value: 652
Transaction timestamp: 13
65
Sender Name: D
Receiver Name: W
Transaction Value: 858
Transaction timestamp: 12
66
Sender Name: B
Receiver Name: X
Transaction Value: 550
Transaction timestamp: 11
67
Sender Name: B
Receiver Name: W
Transaction Value: 850
Transaction timestamp: 10
68
Sender Name: D
Receiver Name: Z
Transaction Value: 454
Transaction timestamp: 9
69
Sender Name: C
Receiver Name: X
Transaction Value: 64
Transaction timestamp: 8
70
Sender Name: C
Receiver Name: W
Transaction Value: 730
Transaction timestamp: 7
71
Sender Name: D
Receiver Name: X
Transaction Value: 373
Transaction timestamp: 6
72
Sender Name: C
Receiver Name: Y
Transaction Value: 80
Transaction timestamp: 5
73
Sender Name: A
Receiver Name: W
Transaction Value: 45
Transaction timestamp: 4
74
Sender Name: D
Receiver Name: X
Transaction Value: 534
Transaction timestamp: 3
75
Sender Name: C
Receiver Name: Y
Transaction Value: 351
Transaction timestamp: 2
76
Sender Name: A
Receiver Name: W
Transaction Value: 453
Transaction timestamp: 1